modules/extensions-spring-support/src/test/resources/dynamic-config-expiration.xml

```diff
@@ -8,7 +8,7 @@
       http://www.mulesoft.org/schema/mule/test http://www.mulesoft.org/schema/mule/test/current/mule-test.xsd">
 
     <configuration>
-        <dynamic-config-expiration frequency="1000" timeUnit="MILLISECONDS">
+        <dynamic-config-expiration frequency="500" timeUnit="MILLISECONDS">
```

Comment:
why reduce this? Can't this make the test flaky when running in jenkins?

Reply:
It is done to detect expired configs faster, otherwise expiration times need to be longer in order to accomodate for the case where it just polled and it needs to wait another full second for the re-check (this in turn makes the whole test suite take longer). Pretty sure it does not affect any test, and even if it does it would mean that the test was wrong or flaky.